tests/test_plugin_loader.py

```
@@ -1812,6 +1831,172 @@ def test_load_plugins_opted_in_commit_unchanged_skips_dependency_install(
+    @patch("mmrelay.plugin_loader.get_community_plugin_dirs")
+    @patch("mmrelay.plugin_loader.get_custom_plugin_dirs")
+    @patch("mmrelay.plugin_loader.start_global_scheduler")
+    @patch("mmrelay.plugin_loader._check_commit_pin_for_upstream_updates")
+    def test_load_plugins_matching_commit_empty_deps_reinstalls_requirements(
+        self,
+        mock_update_check,
+        mock_start_scheduler,
+        mock_get_custom_dirs,
+        mock_get_community_dirs,
+        mock_load_from_dir,
+        mock_resolve_local_head_commit,
```

Ruff (0.15.11)

[warning] Unused method argument: `mock_resolve_local_head_commit`

(ARG002)